src/coreclr/src/jit/lower.h

```diff
@@ -317,6 +317,139 @@ class Lowering final : public Phase
     void LowerHWIntrinsicCC(GenTreeHWIntrinsic* node, NamedIntrinsic newIntrinsicId, GenCondition condition);
     void LowerHWIntrinsicCreate(GenTreeHWIntrinsic* node);
     void LowerFusedMultiplyAdd(GenTreeHWIntrinsic* node);
+
+    union VectorConstant {
```

Comment:
This was just moved from https://github.com/dotnet/runtime/pull/36267/files#diff-55de8576070c2b77ed274e8e03c6676fL1103